emperor/support_files/js/sceneplotview3d.js

```diff
@@ -100,6 +103,26 @@ define([
     this.dimensionRanges = {'max': [], 'min': []};
     this.drawAxesWithColor(0xFFFFFF);
     this.drawAxesLabelsWithColor(0xFFFFFF);
+
+    this.raycaster = new THREE.Raycaster();
+    this.mouse = new THREE.Vector2();
```

Comment:
Noted, thanks! Can you add this as a "private" property then?

On (May-03-16|13:03), Joshua Shorenstein wrote:
> > @@ -100,6 +103,26 @@ define([
> >      this.dimensionRanges = {'max': [], 'min': []};
> >      this.drawAxesWithColor(0xFFFFFF);
> >      this.drawAxesLabelsWithColor(0xFFFFFF);
> > +
> > +    this.raycaster = new THREE.Raycaster();
> > +    this.mouse = new THREE.Vector2();
> 
> The suggestion from [the docs](http://threejs.org/docs/#Reference/Core/Raycaster) and [stackoverflow](http://stackoverflow.com/a/18553739) suggest just instantiating it once and reusing it.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/467/files/a49ad1a3f02e936b9e50a101a620a229a8d6a9c1#r61946089